test/quadratic.jl

```diff
@@ -49,7 +49,7 @@ xlo, xhi = itp3[.2], itp3[xmax+.7]
 @test_approx_eq xhi A[end]
 
 # Check continuity
-xs = [0:.1:length(A)+1]
+xs = [0:.1:length(A)+1;]
```

Comment:
`vcat(rng)` or `convert(Vector{Float64}, rng)` are also choices; which do you prefer?